src/Avalonia.Controls/Platform/ITopLevelImpl.cs

```diff
@@ -38,6 +39,18 @@ public interface ITopLevelImpl : IDisposable
         IPlatformHandle Handle { get; }
```

Comment:
Handle is still used by Direct2D. It's also utilized by embedding and designer code. But the main purpose of still having it is that people will still need to get HWND/XID/whatever in their applications.